.github/workflows/stale-issues.yml

```
@@ -20,10 +20,10 @@ jobs:
         with:
           stale-issue-message: 'Ping! There has been no activity for 7 days.'
           stale-pr-message: 'Ping! There has been no activity for 7 days.'
-          only-label: 'status: awaiting answer/followup'
+          only-labels: 'status: awaiting answer/followup'
           days-before-stale: 7
           days-before-close: -1
-          stale-issue-label: ''
-          stale-pr-label: ''
+          stale-issue-label: 'Stale'
+          stale-pr-label: 'Stale'
```

I just made a proper label in which you should use.
```suggestion
          stale-issue-label: 'status: stale'
          stale-pr-label: 'status: stale'
```

---

I'm not sure what the behavior of the action is when the label doesn't exist.
I would prefer if no tag was added.

---

Based on my knowledge, if there is no such label, it would just create the label beforehand, by force. I could delete the `status: stale` label if you want to test this hypothesis, but I would still like to keep it as a fallback, if what I think is right.

---

Sure, let's keep it